docs/operators/transform.md

```diff
@@ -0,0 +1,121 @@
+Transform - Operator Documentation
+==================================
+
+About Transform operator
+------------------------
+
+Transform means mapping of field expression from input to output or conversion of fields from one type to another.
```

Comment:
General observation about writing docs for operators: We can assume the reader is familiar with Java and related terminology but we cannot assume much more than that. In particular, terms and phrases that have a specialized meaning in the Apex context must be explained (e.g. "expression mappings"). Secondly the class name should be mentioned early on so that people can find and view the code in conjunction with this document.

In this particular instance, I would suggest rewriting this paragraph like this:

This operator receives objects on its input port; for each such input object, it creates
a new output object whose fields are computed as expressions involving fields of the input
object. The types of the input and output objects are configurable as are the expressions
used to compute the output fields.

The operator class is `TransformOperator` located in the package `com.datatorrent.lib.transform`.